docs/tutorials/getting-started.md

```diff
@@ -0,0 +1,118 @@
+hello-kubecon/0*      active    idle   10.1.97.193   
+```
+
+This means that Content-cache-k8s charm isn't integrated with Hello-kubecon yet.
+
+### Relate to the Hello-kubecon charm
+
+Provide integration between Content-cache-k8s and Hello-kubecon by running the following [`juju relate`](https://juju.is/docs/olm/juju-relate) command:
+
+```bash
+juju relate content-cache-k8s:ingress-proxy hello-kubecon
+```
+
+Run `juju status` to see that the message has changed:
+
+```bash
+content-cache-k8s/0*  active    idle   10.1.97.227         Ready
+hello-kubecon/0*      active    idle   10.1.97.193
+```
+
+Note: `ingress-proxy` is the name of the relation. This is needed because establishes that the two charms are compatible with each other.  You can run `juju info content-cache-k8s` to check what are the relation names provided by the Content-cache-k8s application.
+
+The deployment finishes when the status shows "Active".
+
+### Relate to Ingress by using NGINX Ingress Integrator
+
+The NGINX Ingress Integrator charm can deploy and manage external access to HTTP/HTTPS services in a Kubernetes cluster.
+
+If you want to make Content-cache-k8s charm available to external clients, you need to deploy the NGINX Ingress Integrator charm and integrate Content-cache-k8s with it.
+
+See more details in [Adding the Ingress Relation to a Charm](https://charmhub.io/nginx-ingress-integrator/docs/adding-ingress-relation).
+
+Deploy the charm NGINX Ingress Integrator:
+
+```bash
+juju deploy nginx-ingress-integrator
+```
+
+If your cluster has [RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) enabled, you'll be prompted to run the following:
+
+```bash
+juju trust nginx-ingress-integrator --scope cluster
+```
+
+Run `juju status` to verify the deployment.
+
+Provide integration between Content-cache-k8s and NGINX Ingress Integrator:
+
+```bash
+juju relate content-cache-k8s:ingress nginx-ingress-integrator
+
+```
+
+To see the Ingress resource created, run `kubectl get ingress` on a namespace named for the Juju model you've deployed the Content-cache-k8s charm into. The output is similar to the following:
+
+```bash
+NAME                    CLASS    HOSTS           ADDRESS     PORTS   AGE
+hello-kubecon-ingress   public   hello-kubecon   127.0.0.1   80      2m11s
+```
+
+Run `juju status` to see the same Ingress IP in the `nginx-ingress-integrator` message:
+
+```bash
+nginx-ingress-integrator                                active      1  nginx-ingress-integrator  stable    45  10.152.183.233  no       Ingress IP(s): 127.0.0.1, Service IP(s): 10.152.183.66
+```
+
+The browser uses entries in the /etc/hosts file to override what is returned by a DNS server.
+
+The default hostname for the Hello-kubecon application is `hello-kubecon`. To resolve it to your Ingress IP, edit [`/etc/hosts`](https://manpages.ubuntu.com/manpages/kinetic/man5/hosts.5.html) file and add the following line accordingly:
```

Comment:
How about rather than "To resolve it to your Ingress IP, edit [`/etc/hosts`](https://manpages.ubuntu.com/manpages/kinetic/man5/hosts.5.html) file and add the following line accordingly:" having "To be able to browse to this site, that hostname will need to resolve to the IP address of your Ingress. You can achieve this by editing [`/etc/hosts`](https://manpages.ubuntu.com/manpages/kinetic/man5/hosts.5.html) file and adding the following line:"?